# Single nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's Disease

The gene-regulatory landscape of the brain is highly dynamic in health and disease, coordinating a menagerie of biological processes across distinct cell-types. Understanding these regulatory programs requires a holistic experimental and analytical approach. Here, we present a multi-omic single-nucleus study of 191,890 nuclei in late-stage Alzheimerâ€™s Disease (AD), profiling chromatin accessibility and gene expression in the same biological samples and uncovering vast glial heterogeneity. We identified cell-type specific, disease-associated candidate cis-regulatory elements and their candidate target genes, including an oligodendrocyte- associated regulatory module containing links to APOE and CLU. We describe cis-regulatory relationships in specific cell-types at AD risk loci defined by genome wide association studies (GWAS), demonstrating the utility of this multi-omic single-nucleus framework for uncovering disease and cell-type-specific regulatory mechanisms. Trajectory analysis of glial populations highlighted transcription factors dysregulated in disease-associated glia, and we identified disease-relevant regulatory targets of these transcription factors. Further, we introduce scWGCNA, a co-expression network analysis strategy robust to the sparsity of single-cell data, to perform a systems-level meta-analysis of AD transcriptomics. Our analyses altogether present SREBF1 as a novel target of interest for AD research, specifically in oligodendrocytes. Finally, this work is highly accessible through our intuitive web portal, allowing for straightforward interrogation of this multi-omic dataset.
